web/admin/src/AttendeeSelector.js

```diff
@@ -133,7 +133,7 @@ export default class AttendeeSelector extends PureComponent {
 
   menuHeaderText = () => {
     const {attendeeIds, tierIds, groupIds} = this.props.content
-    if (groupIds.length) return 'Filters selected'
+    if (groupIds.length) return groupIds.length + ' Filters selected'
```

Comment:
This is not necessarily the correct number of filters selected.  There may also be tiers and individual attendees selected.  `Filters selected` is intentionally vague, since we don't have any idea how many attendees are selected.